ops/rak_button_firmware.ahk

```diff
@@ -0,0 +1,184 @@
+; Created 2022-07
+; by Peter Scholtens and Kerry Wang
+; Instructions can be found here: https://app.clickup.com/2434616/v/dc/2a9hr-2261/2a9hr-6207
+; The instructions for compiling are at the bottom of the page
+
+#NoEnv  ; Recommended for performance and compatibility with future AutoHotkey releases.
+; #Warn  ; Enable warnings to assist with detecting common errors.
+SendMode Input  ; Recommended for new scripts due to its superior speed and reliability.
+SetWorkingDir %A_ScriptDir%  ; Ensures a consistent starting directory.
+name := "REPLACENAME"
+counter := 4444
+
+Clear() {
+    SendInput ^a
+    SendInput {delete}
+}
+
+PressOpenPort() {
+    MouseClick, left, 513, 68
+}
+
+PressTxtBox() {
+    MouseCLick, left, 284, 934
+}
+
+PressEnter() {
+    MouseCLick, left, 1285, 940
+}
+
+PressEui() {
+    MouseClick, left, 162, 308
+}
+
+PressScrollUp() {
+    MouseClick, left, 1315, 131
+}
+
+ClickAwsAddDevice() {
+    MouseClick, left, 1660, 440
+}
+
+
+; Command to set heartbeat and get DEVEUI
+!^+a::
+Sleep, 500
+Run, C:\Users\dev\OneDrive\Desktop\RAK_SERIAL_PORT_TOOL_V1.2.1\RAK_SERIAL_PORT_TOOL_V1.2.1.exe
+Sleep, 500
+SendInput #{Up}
+Sleep, 500
+
+; in-app commands
+PressOpenPort()
+Sleep, 500
+
+; set heartbeat
+PressTxtBox()
+Sleep, 500
+Clear()
+Sleep, 500
+SendInput at{+}heartbeat=1
+Sleep, 500
+PressEnter()
+Sleep, 500
+
+; check status
+PressTxtBox()
+Sleep, 500
+Clear()
+Sleep, 500
+SendInput at{+}get_config=lora:status
+Sleep, 500
+PressEnter()
+Sleep, 500
+PressEui()
+PressEui()
+SendInput ^c
+Sleep, 500
+
+; google sheets paste eui
```

Comment:
How does it know which Google Sheet to use?

Reply:
The script assumes that the Google Sheets is tab open on the only instance of Google Chrome.

Reply:
OK, yup, I see that in the docs now.